setup.py

```diff
@@ -64,11 +69,11 @@
     sys.exit(-1)
 
 setup(
-    version              = '4.12.0dev',
```

Comment:
Please avoid unrelated style changes. I don't think stripping the alignment makes this code more readable.